### PR TITLE
Add Portland 2023 draft schedule

### PR DESCRIPTION
**Pull request by mxsasha**
----
:books: Documentation preview :books:: https://writethedocs-www--1894.org.readthedocs.build/

- [ ] We should find some consistent emoji or something, probably? 
- [x] And we need to figure out how to fit unconf slots into this new schedule
- [x] Probably need a Monday afternoon coffee break? 
- [x] Will need a blog post

### CODE REVIEW

**Comment by ericholscher** [comment]
Updated copy to note on-stage Q&A :)

**Comment by plaindocs** [comment]
I'll add current names to this, and dust it off. 

Anyone know anything about this visual glitch? It feels new to me. 

image

**Comment by plaindocs** [comment]
I'd also like to know why the spellcheck is suddenly saying this is error from April 2021:

```
4ac322aa4 (Barrie Byron     2021-04-26 11:48:04 -0400 226) - [The art of writing Release Notes A guide, examples & template -- Slite](https://slite.com/learn/release-notes)
```

File: docs/_templates/include/schedule2021.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.. while called schedule2021, this file applies to 2021 *and newer*
```
Comment (plaindocs) [inline docs/_templates/include/schedule2021.rst:1]: What were the people who designed this hack thinking :japanese_ogre: shall we fix?

**Comment by canncrochet** [comment]
Looks great, thank you!

**Comment by ericholscher** [comment]
> Anyone know anything about this visual glitch? It feels new to me.

@plaindocs That's my fault -- I wanted to add the CFP to the menu, and copied the styling 🙃. I don't think it should be happening in `main`, so might just need to update the base.

**Review by plaindocs — APPROVED** [review]

File: docs/conf/portland/2023/news/announcing-schedule.rst [hunk 2]
```diff
@@ -12,45 +12,52 @@ The {{city}} conference is less than two months away, and we're busy making thin
...
-We're learning and adapting from our virtual events from the past couple of years, so we're introducing a moderated Q&A session after each speaker, with exact details of how that will work still to be confirmed.
+Join us for our hike again this year
+------------------------------------
 
-Hiking!
--------
+We're reviving the traditional Pittock Mansion `hike <https://www.writethedocs.org/conf/portland/{{year}}/outing/>`_ on Saturday. This is a great way to experience the beauty that Portland holds within city limits, with a walk along a creek, a majestic view of Mount Hood, and finishing with a meader through Hoyt Arboritum with hundreds of different trees and plants. 
 
-We're reviving the traditional Pittock Mansion `hike <https://www.writethedocs.org/conf/portland/{{year}}/outing/>`_ on Saturday, keep an eye on future emails for details on exactly where and when we're meeting.
+The hike will be on  **{{ hike.date }} **. Please register a `free ticket <https://ti.to/writethedocs/write-the-docs-portland-2023/with/hike-ticket>`_ so we can contact you with any last minute weather and logistics information.
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/announcing-schedule.rst]:
```suggestion
The hike will be on  **{{ hike.date }}**. Please register a `free ticket <https://ti.to/writethedocs/write-the-docs-portland-2023/with/hike-ticket>`_ so we can contact you with any last minute weather and logistics information.
```

Hopefully that'll shut the linter warning below up?

File: docs/conf/portland/2023/news/announcing-schedule.rst [hunk 3]
```diff
@@ -12,45 +12,52 @@ The {{city}} conference is less than two months away, and we're busy making thin
...
 
 Start thinking about Writing Day
 --------------------------------
 
-Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/{{year}}/writing-day/>`_ is happening on Sunday. It is much more fun when folks come with projects in mind!
+Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/{{year}}/writing-day/>`_ is happening on Sunday. It's a great chance to learn from other attendees, and get some contributors on a project you have in mind. There is also room for having discussions and showing off fun new tools you've found to folks.
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/announcing-schedule.rst]:
```suggestion
Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/{{year}}/writing-day/>`_ is happening on Sunday. It's a great chance to learn from other attendees, and get some contributors on a project you have in mind. There is also room for having discussions and showing off fun new tools you've found.
```

**Pull request merged**